dependencies.gradle

```diff
@@ -1,9 +1,9 @@
 ext.versions = [
         agp                    : '7.0.3',
-        compileSdk             : 30,
-        targetSdk              : 30,
+        compileSdk             : 31,
+        targetSdk              : 31,
```

Comment:
It also updates compile + target SDK to `31`(then some other DependABot created PRs should compile)